habitat-lab/habitat/config/habitat/task/rearrange/cat_nav_to_obj.yaml

```diff
@@ -42,6 +42,7 @@ constraint_violation_drops_object: True
 object_in_hand_sample_prob: 0.0
 goal_type: ovmm
 camera_tilt: -0.5236
+receptacle_categories_file: data/objects/fpmodels-categories.csv
```

Comment:
Should this be under `data/objects` or `data/fphab/fphab/semantics`?

Reply:
There are a bunch of these files that resulted from our annotations, so I've put them in `data/objects` for now.